index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
@ianbjacobs - I'm still not clear what "capabilities provided by data" means. It doesn't make sense to me.

The spec should be talking about what is in the `data` as "filtering" the available payment handlers (even if not called "filters" explicitly). How about:

"Determine which payment handlers support identifier. Filter the set of matching payment handlers further by comparing the capabilities of each payment handler with the content of `data`. For each payment handler; if `data` implies that payment method specific capabilities are required and these are not provided, the payment handler does not match, otherwise the payment handler matches."

Bare in mind that we are exploring a `canMakePayment()` lambda on handlers so the assumption is that this would be called here for web-based handlers BUT we don't (can't) talk about that explicitly and create a dependency on Payment Handler so we need to have something that implies that (or similar behavior) but is generic enough to not introduce the dependency and also works for other types of handler like the built in handler for basic card and platform native handlers too.